indigo/distribution.yaml

```diff
@@ -5368,6 +5368,10 @@ repositories:
       url: https://github.com/Oriense/orsens_ros.git
       version: master
   p2os:
+    doc:
+      type: git
+      url: https://raw.githubusercontent.com/allenh1/p2os/master/p2os_doc/p2os_driver/html/index.html
```

Comment:
This needs to be a git url not an html url: https://travis-ci.org/ros/rosdistro/builds/60993049#L318